ui/src/components/common/Header.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+
+import {
+  Dropdown,
+  DropdownToggle,
+  DropdownItem,
+  Page,
+  PageHeader,
+  PageSection,
+  PageSectionVariants,
+  Toolbar,
+  ToolbarGroup,
+  ToolbarItem
+} from '@patternfly/react-core';
+import accessibleStyles from '@patternfly/patternfly/utilities/Accessibility/accessibility.css';
+import { css } from '@patternfly/react-styles';
+import './Header.css';
+
+class Header extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isUserDropdownOpen: false,
+      showAboutModal: false
+    };
+
+    this.onTitleClick = this.onTitleClick.bind(this);
+    //this.onLogoutUser = this.onLogoutUser.bind(this);
+
+    this.onUserDropdownToggle = this.onUserDropdownToggle.bind(this);
+    this.onUserDropdownSelect = this.onUserDropdownSelect.bind(this);
+  }
+
+  onTitleClick = () => {
+    const { history } = this.props;
+    history.push('/');
+  };
+
+  onUserDropdownToggle(isUserDropdownOpen) {
+    this.setState({
+      isUserDropdownOpen
+    });
+  }
+
+  onUserDropdownSelect = () => {
+    this.setState({
+      isUserDropdownOpen: !this.state.isUserDropdownOpen
+    });
+  };
+
+  render() {
+    const { isUserDropdownOpen } = this.state;
+
+    const logoProps = {
+      onClick: () => this.onTitleClick()
+    };
+
+    /* const logoProps = {
+      href: 'https://patternfly.org',
+      onClick: () => console.log('clicked logo'),
+      target: '_blank'
+    }; */
+
+
+    const toolbar = (
+      <>
```

Comment:
I'm not familiar with the use of `<>`. Is this a PF thing?

Reply:
A React thing actually - it's shorthand for [`<React.Fragment>`](https://reactjs.org/docs/fragments.html). Must see if it's actually needed here though

Reply:
@damienomurchu given that you both open and close the Fragment with Toolbar, I'm pretty sure that you don't need it.